twake/backend/node/src/core/platform/services/tracker/index.ts

```diff
@@ -108,7 +109,7 @@ export default class Tracker extends TwakeService<TrackerAPI> implements Tracker
   ): Promise<void> {
     const analiticsIdentity = {
       userId: identity.user?.allow_tracking ? identity.user.identity_provider_id : undefined,
-      anonymousId: "anonymous",
+      anonymousId: "anonymous-" + md5(identity.user.identity_provider_id),
```

Comment:
``` js
`anonymous-${md5(identity.user.identity_provider_id)}`
```

Reply:
Instead of using anonymousId, could you add `anonymous-${md5(identity.user.identity_provider_id)}` to the userId property ? :)

Reply:
Ok, why is the anonymousId not usable in this case, isn't it the good way to do it usually ? (just curious)

Reply:
Segment can handle anonymousId. I guess is more used for landing analytics. 
But Amplitude can't work without userId :)